packages/nuxt-locokit/src/pages/workspace/index.vue

```diff
@@ -64,18 +62,19 @@
 <script setup lang="ts">
 import PrimeButton from 'primevue/button'
 import { useI18n } from 'vue-i18n'
+import { storeToRefs } from 'pinia'
 import { ROUTES_NAMES } from '../../paths'
 import { useStoreWorkspaces } from '../../stores/workspaces'
 import WithBanner from '../../layouts/WithHeader.vue'
-import { useHead, onMounted } from '#imports'
+import { useHead } from '#imports'
 
 const { t } = useI18n({ useScope: 'global' })
 
 const workspacesStore = useStoreWorkspaces()
+const { workspaces } = storeToRefs(workspacesStore)
 
-onMounted(async () => {
-  await workspacesStore.findWorkspaces()
-})
+// Initialization
+await workspacesStore.updateWorkspaces()
```

Comment:
I call it like that because this function update store but that may not relevant…

Indeed, there is no pagination yet. I didn't do it because we didn't agree on the representation between workspaces belonging to the logged-in user, member workspaces and public workspaces.

Reply:
Oh, that's interesting indeed, maybe `workspacesStore.update()` could me more relevant ?

Reply:
Currently in the store there are a ref for the currentWorkspace and another ref for workspaces. So I need to distinguish them when I updated

Reply:
Ok